### ISSUE TITLE
Please make a release

### ISSUE THREAD

**Issue by yurivict** [opendtrace/toolkit#10]
The FreeBSD port is at version 1.0, but no release has ever been made here.
 Could you please make a release 1.0 to match the port?

**Comment by athompso** [comment]
@gnn will have the final say on this, but that's not how port versioning and source versioning interact with each other.  My use of "we" is rhetorical, I'm not a maintainer, just an interested third party.

This github project has no releases at all, which means FreeBSD can version the port whatever they want.
Since some of the code tracks Solaris releases, shouldn't we instead match their versioning?
Oh, wait, other bits track MacOS X, so maybe we need to track their versioning?

Bottom line: I think you've misunderstood how multi-OS projects do versioning, sorry.

**Comment by yurivict** [comment]
No, FreeBSD has a policy that versioning should be based on the upstream version. If there is no upstream version, then they just set the version to the date of the latest change, ex. g20180319. 'g' means that such version isn't incremental or comparable. FreeBSD doesn't make version up for the port.

So if there is no plan to ever version it, I should reset the port version to this 'g' scheme.

**Comment by athompso** [comment]
Not my call, but as someone who uses this on multiple OSes (*not* including Freebase) I really don't see how ever assigning a version number to this would ever make sense.
IMHO I'd say wait for a maintainer to comment, otherwise revert to the git/date scheme.

**Comment by gvnn3** [comment]
For now let's stick to git/date scheme.  I'll cut a release soon but I can't commit to a date.
